src/exec.sh

```diff
@@ -107,7 +105,10 @@ cd build-${TIMESTAMP} # IN BUILD DIRECTORY =============================
 tar -xzf ../uploads/$TARBALL
 testIt
 tar -cf ../screenshots/${TIMESTAMP}.tar "$TESTS_FOLDER/screenshots"
-cd - # IN ROOT DIRECTORY ==================================
+
+# CD TO ROOT DIRECTORY ==================================
+cd $DIR/..
+
```

Comment:
Yes... `testit` had to get into the dist directory to run the http-server. Previously, we ran http-server in the base directory, and the browser had to visit `http://localhost:xyzw/demo`.

I wanted to make this url consistent across different types of apps.